DistroLauncher/Patch.cpp

```diff
@@ -44,6 +44,49 @@ namespace Ubuntu::PatchingFunctions
         conf << "[Unit]\nConditionVirtualization=!container\n";
         return !conf.fail();
     }
+
+    bool SetDefaultUpgradePolicy(std::istreambuf_iterator<char> unused, std::ostream& output)
```

Comment:
A minor thing: `unused` is being used, so we should assign a more meaningful name.